scripts/fast-reboot

```diff
@@ -715,6 +739,9 @@ if [ -x ${DEVPATH}/${PLATFORM}/${PLATFORM_PLUGIN} ]; then
     ${DEVPATH}/${PLATFORM}/${PLATFORM_PLUGIN}
 fi
 
+# Re-enable swss autorestart for the next boot
+config feature autorestart swss enabled
```

Comment:
@jleveque in this step my intention was to re-enable autorestart of `swss`. However, in my tests, I have noticed that after warm-reboot `swss autorestart` remains disabled.
Do you know what am I doing missing here?

Reply:
I can only assume the saved configuration has it marked as disabled? But I see you're not saving the config after disabling it, so it's not you doing it. But I would start by checking that, and if so, figure out why it's saved as disabled.

Reply:
I checked again, and this was due to database container being killed/stopped before that, therefore it was not able to enable it back
```
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/redis/connection.py", line 559, in connect
    sock = self._connect()
  File "/usr/local/lib/python3.7/dist-packages/redis/connection.py", line 615, in _connect
    raise err
  File "/usr/local/lib/python3.7/dist-packages/redis/connection.py", line 603, in _connect
    sock.connect(socket_address)
ConnectionRefusedError: [Errno 111] Connection refused
```

Further, I found out that after executing `systemctl stop swss`, the container do not automatically restart even if autorestart is enabled for swss at the time.
In comparison, for `201811` branch, after `docker kill swss`  monit/systemctl timers keeps restarting swss